src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
 body {
   margin: 0;
+  font-family: Roboto, serif;
+}
+
+p,
+h1,
+h2,
+h3 {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+img {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+h1 {
+  color: #060b35;
+  margin-bottom: 4px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+p {
+  font-weight: 400;
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('/src/images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('/src/images/star-active.svg');
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 0;
+}
+
+.Price {
```

Comment:
The class name 'Price' should follow a consistent naming convention. Consider using lowercase, such as 'price', to match the other class names like 'card__stars' and 'card__reviews'. This will ensure consistency with the HTML file as well.